test/suite/test_layered27.py

```diff
@@ -57,46 +66,161 @@ def test_drain_insert(self):
         session_follow = conn_follow.open_session('')
         session_follow.create(self.uri, "key_format=S,value_format=S")
 
-        # Create some oplog traffic, a mix of inserts and updates
-        oplog.insert(t, 100)
+        # Create some oplog traffic
+        oplog.insert(t, 100 * self.multiplier)
 
         # Apply them to leader WT and checkpoint.
-        oplog.apply(self, self.session, 0, 100)
-        oplog.check(self, self.session, 0, 100)
+        oplog.apply(self, self.session, 0, 100 * self.multiplier)
+        oplog.check(self, self.session, 0, 100 * self.multiplier)
 
         self.conn.set_timestamp(f'stable_timestamp={self.timestamp_str(oplog.last_timestamp())}')
 
         self.session.checkpoint()     # checkpoint 1
 
         # Add some more traffic
-        oplog.insert(t, 100)
-        oplog.update(t, 200)
-        oplog.apply(self, self.session, 100, 300)
-        oplog.check(self, self.session, 0, 400)
+        oplog.insert(t, 100 * self.multiplier)
+        oplog.update(t, 200 * self.multiplier)
+        oplog.apply(self, self.session, 100 * self.multiplier, 300 * self.multiplier)
+        oplog.check(self, self.session, 0, 400 * self.multiplier)
+
+        # On the follower -
+        # Apply all the entries to follower
+        oplog.apply(self, session_follow, 0, 400 * self.multiplier)
+
+        self.pr(f'OPLOG: {oplog}')
+        oplog.check(self, session_follow, 0, 400 * self.multiplier)
+
+        # Then advance the checkpoint and make sure everything is still good
+        self.pr('advance checkpoint')
+        self.disagg_advance_checkpoint(conn_follow)
+        oplog.check(self, session_follow, 0, 400 * self.multiplier)
+
+        self.conn.close()
+        conn_follow.reconfigure('disaggregated=(role="leader")')
+
+        # Checkpoint after draining the ingest table
+        conn_follow.set_timestamp(f'stable_timestamp={self.timestamp_str(oplog.last_timestamp())}')
+        session_follow.checkpoint()
+
+        # Reopen the new leader to get rid of the content in the ingest table
```

Comment:
Shouldn't comment say reopening the new _follower_? instead of leader.

Reply:
fixed